packages/dds/merge-tree/src/test/Removal.perf.spec.ts

```diff
@@ -61,6 +63,46 @@ describe("MergeTree remove", () => {
 		},
 	});
 
+	for (const length of [10, 100, 1000]) {
+		benchmark({
+			type: BenchmarkType.Measurement,
+			title: "remove large range of large tree with undo-redo",
+			category: "remove",
+			before: () => {
+				const str = new TestString("id", {});
+				for (let i = 0; i < length / 2; i++) {
+					str.append("a", true);
+					str.appendMarker(true);
+				}
+
+				str.applyPendingOps();
+				summary = str.getSummary();
+			},
+			benchmarkFnAsync: async () => {
+				const str = await loadSnapshot(summary);
+				const driver = createRevertDriver(str);
+
+				const revertibles: MergeTreeDeltaRevertible[] = [];
+				str.on("delta", (_op, delta) => {
+					appendToMergeTreeDeltaRevertibles(driver, delta, revertibles);
+				});
+
+				// TODO: deal with end of tree segment too.
```

Comment:
should probably just file some work on this and remove the comment. doesn't need to be fixed here.